app/Models/Certificate.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Certificate extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'lecturer_id',
+        'subject',
+        'publisher',
+        'date',
+    ];
+
+    public function lecturerId()
```

Comment:
nama fungsi itu yang merepresentasikan nama model yang direlasi, bukan kolomya
```suggestion
    public function lecturer()
```